sdgym/synthesizers/__init__.py

```diff
@@ -5,7 +5,7 @@
 from sdgym.synthesizers.medgan import MedGAN
 from sdgym.synthesizers.privbn import PrivBN
 from sdgym.synthesizers.sdv import (
-    CTGAN, CopulaGAN, GaussianCopulaCategorical, GaussianCopulaCategoricalFuzzy,
+    CTGAN, TVAE, CopulaGAN, GaussianCopulaCategorical, GaussianCopulaCategoricalFuzzy,
```

Comment:
Thanks @tejuafonja! It looks like we're also missing imports for `HMA1` and `PAR` from `sdgym.synthesizers.sdv`. Could you add those for completeness?

Reply:
Thanks for reviewing this PR @katxiao. I've added the missing imports.

Reply:
Hi @tejuafonja, thanks for addressing! Could you also run `make fix-lint`? It looks like the imports are incorrectly sorted, and `make fix-lint` will fix it. You can check if the lint passes by running `make lint`.

Reply:
Thanks, @katxiao I've correctly sorted the imports and created another PR #140. Sorry, I couldn't figure out how to link it to this PR since I had to clone the repo to run the tests.